src/sentry/api/endpoints/organization_details.py

```diff
@@ -248,6 +288,55 @@ def validate(self, attrs):
                 )
         return attrs
 
+    def save_trusted_relays(self, incoming, changed_data, organization):
+        timestamp_now = datetime.utcnow().replace(tzinfo=UTC).isoformat()
+        option_key = "sentry:trusted-relays"
+        try:
+            # get what we already have
+            existing = (
+                OrganizationOption.objects.get(organization=organization, key=option_key) or {}
```

Comment:
I must have overlooked this in my previous review when I suggested to coerce `existing` to an empty tuple. For some reason, I thought that this is just the plain value of the option (which is a list). Instead, this is an object with attributes. 

That said since we will get into the `except` case, we should remove this `or {}` since it would never work.

```suggestion
                OrganizationOption.objects.get(organization=organization, key=option_key)
```

Reply:
yes or {} makes no sense

Reply:
done